src/checkers/inference/DefaultSlotManager.java

```diff
@@ -577,14 +577,11 @@ public ExistentialVariableSlot createExistentialVariableSlot(Slot potentialSlot,
 
     /**
      *  Determine the type kind of an arithmetic operation, based on Binary Numeric Promotion in JLS 5.6.2.
-     * @param lhsAtm atm of left operand
-     * @param rhsAtm atm of right operand
+     * @param lhsType type of left operand
```

Comment:
Could you break this PR into two, one for compound assignments and one for these parameter type changes?